ops/terraform/services/base/values/prod.yaml

```diff
@@ -108,7 +108,7 @@
 /bfd/${env}/common/nonsensitive/rds_master_username: bfduser
 /bfd/${env}/common/nonsensitive/rds_cluster_identifier: bfd-prod-aurora-cluster
 /bfd/${env}/common/nonsensitive/rds_instance_class: db.r6i.12xlarge
-/bfd/${env}/common/nonsensitive/rds_instance_count: "4"
+/bfd/${env}/common/nonsensitive/rds_instance_count: "3"
```

Comment:
straightforward change - would like to understand process of applying this change

Reply:
As of 20 June 2024 22:34:40 GMT the value on line 111 matches the value in AWS ... ?

Reply:
> straightforward change - would like to understand process of applying this change

It is! For path-to-production **and** ephemeral environments, both application and terraservice-specific changes like this are applied as part of the deployment process _sequentially_. This change impacts the `base` terraservice which is part of the first stage in the series of deployment stages. For example, see the `base` deployment logic in `test` [here in the Jenkinsfile](https://github.com/CMSgov/beneficiary-fhir-data/blob/888330ffe59de67dc32fcdd9c1e81620399539fa/Jenkinsfile#L216-L217).

> As of 20 June 2024 22:34:40 GMT the proposed new value on line 111 matches the value in AWS ... ?

See BFD-3497 _or_ [my contributions team sync slack thread from 2024-06-20](https://cmsgov.slack.com/archives/C010WDXAZFZ/p1718898949977479?thread_ts=1718893812.464939&cid=C010WDXAZFZ) for more context–this change has already been applied out-of-band and this PR updates the stored infrastructure-as-code to match the current desired state. A change like this will help us avoid _undoing_ those out-of-band changes as part of subsequent deployments.